tasks/windows.ps1

```diff
@@ -0,0 +1,82 @@
+[CmdletBinding()]
+param(
+  # NOTE: init.json cannot yet be shared, so must have windows.json / windows.ps1
+  [Parameter(Mandatory = $true)]
+  [String]
+  $Name,
+
+  [Parameter(Mandatory = $true)]
+  [ValidateSet('start', 'stop', 'restart', 'status')]
+  [String]
+  $Action
+)
+
+$ErrorActionPreference = 'Stop'
+
+function Invoke-ServiceAction($Service, $Action)
+{
+  $inSyncStatus = 'in_sync'
+  $status = $null
+
+  switch ($Action)
+  {
+    'start'
+    {
+      if ($Service.Status -eq 'Running') { $status = $InSyncStatus }
```

Comment:
What about if it's in StartPending?

Reply:
Given the current specification that does not match `Running`

This is one of the reasons I think that we should rethink the responses, removing the `in_sync` status. Better to give the initial service state and the service state after the `action` is performed and let the caller exercise it's own logic around how to surface that to users.